docs/content/_navigation.json

```diff
@@ -123,7 +123,21 @@
         "children": [
           {
             "title": "Partitions",
-            "path": "/concepts/partitions-schedules-sensors/partitions"
+            "path": "/concepts/partitions-schedules-sensors/partitions",
+            "children": [
+              {
+                "title": "Partitioned assets",
+                "path": "/concepts/partitions-schedules-sensors/partitioned-assets"
+              },
+              {
+                "title": "Partitioned op jobs (Advanced)",
+                "path": "/concepts/partitions-schedules-sensors/partitioned-op-jobs"
+              },
```

Comment:
"/partitioned-assets" has the title called "Partitioned assets and jobs". should the nav be consistent with the page title?

However, i found "Partitioned assets and jobs" and "Partitioned op jobs" a little confusing, as in the "jobs" mention in the asset-specific page feels odd. How about:
- "Partitioning assets" - where assets and asset-based jobs can fit in naturally
- "Partitioning ops (Advanced)" - so we don't try to differentiate "op jobs" vs "asset jobs" vs "jobs", and instead, jobs can become an implementation detail when it comes to usage.
- "Testing partitions" - part of which could apply to both assets and ops

Reply:
I like this; I'll make the changes